index.js

```diff
@@ -19,3 +21,155 @@ links.forEach((n) => n.addEventListener('click', () => {
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/canopy-website',
+    project: 'https://github.com/mumbereh/canopy-website',
+  },
+  {
+    id: '03',
+    src: 'images/1.png',
+    title: 'Facebook 360',
+    description1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/Swampflix',
+    project: 'https://github.com/mumbereh/Swampflix',
+  },
+  {
+    id: '04',
+    src: 'images/2.png',
+    title: 'Uber Navigation',
+    description1: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    description2: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed non risus. Suspendisse lectus tortor, dignissim sit amet, adipiscing nec, ultricies sed, dolor.',
+    live: 'https://mumbereh.github.io/Lambada',
+    project: 'https://github.com/mumbereh/Lambada',
+  },
+];
+
+// Select the card button and the popup container
+const cardBtn = document.querySelector('.card-btn');
+const popupContainer = document.querySelector('.popup');
+
+// Create a function to show the popup
+function showPopup() {
+  // Add a class to the popup container to show it
+  popupContainer.classList.add('show');
+
+  // Remove any existing content from the popup
+  popupContainer.innerHTML = '';
+
+  // Loop through the popups array and create the content for each popup
+  popups.forEach((popup) => {
+    const popupContent = `
+      <div class="popup">
+        <h2 class="title">${popup.title}</h2>
+        <ul class="field">
+          <li class="cano">CANOPY</li>
+          <li class="tag">&#8226; Back End Dev </li>
+          <li class="tag">&#8226; 2015</li>
+        </ul>
+        <img src="${popup.src}" alt="">
+        <div class="pop-para">
+          <p>${popup.description1}</p><br>
+          <p>${popup.description2}</p>
+        </div>
+        <div class="pop-skills">
+       <ul class="trainning2">
+        <li class="other-skills">HTML</li>
+        <li class="other-skills">CSS</li>
+        <li class="other-skills">JavaScript</li>
+        <li id="desk2" class="ruby">Ruby </li>
+        <li id="desk2" class="other-skills" >BootStrap</li>
+    </ul>
+    <a href="https://mumbereh.github.io/First-portfolio">
+    <button type="submit">See source<img class="search" src="icon/popup.png"></button>
+  </a>
+    <a href="https://github.com/mumbereh/First-portfolio">
+    <button type="submit">See source<img class="search" src="icon/popup.png"></button>
```

Comment:
- [ ] While I was testing the popups, I realized that only the first project when clicked is opening the popup. Others are just causing the error which can be seen in the console logs. Please correct these errors and make sure that other projects other than the first one is opening their respective popups 😉 
![image](https://user-images.githubusercontent.com/90222110/233807755-53353732-b25a-4499-825b-ffc1083422f0.png)